mockserver-spring/src/main/java/org/mockserver/spring/MockServerTestCustomizerFactory.java

```diff
@@ -0,0 +1,19 @@
+package org.mockserver.spring;
+
+import java.util.List;
+
+import org.springframework.core.annotation.AnnotatedElementUtils;
+import org.springframework.test.context.ContextConfigurationAttributes;
+import org.springframework.test.context.ContextCustomizer;
+import org.springframework.test.context.ContextCustomizerFactory;
+
+public class MockServerTestCustomizerFactory implements ContextCustomizerFactory {
+    @Override
+    public ContextCustomizer createContextCustomizer(Class<?> testClass, List<ContextConfigurationAttributes> configAttributes) {
+        final MockServerTest mockServerTest = AnnotatedElementUtils.findMergedAnnotation(testClass, MockServerTest.class);
+        if (mockServerTest != null) {
```

Comment:
You could do
`return mockServerTest == null ? null : new MockServerPropertyCustomizer(mockServerTest.value());`

Reply:
done